### PR TITLE
Add integration test

### PR DESCRIPTION
**Pull request by sutser**

### CODE REVIEW

File: GradientColorPickerTest/IntegrationTest.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Drawing;
+using Messerli.GradientColorPicker;
+using Xunit;
+
+namespace Messerli.GradientColorPickerTest
+{
+    public class IntegrationTest
+    {
+        private const double GoodPasswordEntropy = 40.0;
```
Comment (tautropfli) [inline GradientColorPickerTest/IntegrationTest.cs]: Restrict the scope of these variables

File: GradientColorPickerTest/IntegrationTest.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            .Build();
+
+        [Theory]
+        [MemberData(nameof(GetEntropy))]
+        public void PasswordStrengthColor(double entropy)
+        {
+            var oldVariantColor = GetOldVariantColor(entropy);
+            var newVariantColor = GetNewVariantColor(Convert.ToInt32(entropy));
+
+            Assert.True(newVariantColor.R - oldVariantColor.R <= 1);
```
Comment (tautropfli) [inline GradientColorPickerTest/IntegrationTest.cs]: Use Assert.Equals(double, double, int precision)

File: GradientColorPickerTest/IntegrationTest.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        private const double BaseBrightness = 191.0;
+        private static readonly GradientColorProvider GradientColorProvider = GradientColorBuilder
+            .WithColor(new GradientColorByValue(Color.FromArgb(255, 191, 191), 0))
+            .Add(new GradientColorByValue(Color.FromArgb(255, 255, 191), 20))
+            .Add(new GradientColorByValue(Color.FromArgb(191, 255, 191), 40))
+            .Build();
+
+        [Theory]
+        [MemberData(nameof(GetEntropy))]
+        public void PasswordStrengthColor(double entropy)
```
Comment (tautropfli) [inline GradientColorPickerTest/IntegrationTest.cs]:
```suggestion
        public void NewAlgorithmProducesSameValesAsOldAlgorithm(double entropy)
```

**Review by tautropfli — CHANGES_REQUESTED** [review]

File: GradientColorPickerTest/IntegrationTest.cs (hunk 2 above)
Comment (sutser) [reply GradientColorPickerTest/IntegrationTest.cs]: Addressed in: https://github.com/messerli-informatik-ag/gradient-color-picker/pull/4/commits/df77c75ae0f59c229a6bd4a4657f2ff50155d0ab

File: GradientColorPickerTest/IntegrationTest.cs (hunk 1 above)
Comment (sutser) [reply GradientColorPickerTest/IntegrationTest.cs]: Addressed in: https://github.com/messerli-informatik-ag/gradient-color-picker/pull/4/commits/f437c7f2a47dce035e245c81c81acd0ecea615f3

File: GradientColorPickerTest/IntegrationTest.cs (hunk 2 above)
Comment (sutser) [reply GradientColorPickerTest/IntegrationTest.cs]: Addressed in: https://github.com/messerli-informatik-ag/gradient-color-picker/pull/4/commits/ac8b87f45391e24525fc46845dfd6e0451fcb9b4

Comment (sutser) [reply GradientColorPickerTest/IntegrationTest.cs]: Addressed in: https://github.com/messerli-informatik-ag/gradient-color-picker/pull/4/commits/ea4613b695bdba288b569ae6d7d4335c28503e4f

**Review by tautropfli — APPROVED** [review]

**Pull request merged**